diff -rupN ./CMakeLists.txt ../express-1.5.1-src_patched/CMakeLists.txt
--- ./CMakeLists.txt	2014-10-12 20:26:03.182961939 +0100
+++ ./CMakeLists.txt	2014-10-12 20:24:16.048374586 +0100
@@ -13,7 +13,7 @@ set(CMAKE_CXX_FLAGS_RHDEBINFO "-O3 -g ${
 set(CMAKE_CXX_FLAGS_MINSIZEREL "-Os ${CMAKE_CXX_FLAGS}")
 
 set(CMAKE_BUILD_TYPE Release)
-set(Boost_USE_STATIC_LIBS ON)
+set(Boost_USE_STATIC_LIBS OFF)
 
 find_package(Boost 1.39
     COMPONENTS
@@ -33,10 +33,10 @@ endif(GPERFTOOLS_TCMALLOC_LIB)
 
 find_package(Protobuf)
 if (PROTOBUF_FOUND)
-  include_directories(${Boost_INCLUDE_DIRS} ${PROTOBUF_INCLUDE_DIR} "${CMAKE_CURRENT_SOURCE_DIR}/bamtools/include") 
+  include_directories(${Boost_INCLUDE_DIRS} ${PROTOBUF_INCLUDE_DIR} $ENV{BAMTOOLS_INCLUDE})
 	set(PROTO_INT 1)
 else (PROTOBUF_FOUND)
-  include_directories(${Boost_INCLUDE_DIRS} "${CMAKE_CURRENT_SOURCE_DIR}/bamtools/include") 
+  include_directories(${Boost_INCLUDE_DIRS} $ENV{BAMTOOLS_INCLUDE})
 	set(PROTO_INT 0)
 endif(PROTOBUF_FOUND)
 
diff -rupN ./src/CMakeLists.txt ../express-1.5.1-src_patched/src/CMakeLists.txt
--- ./src/CMakeLists.txt	2014-10-12 20:26:55.442761032 +0100
+++ ./src/CMakeLists.txt	2014-10-12 20:24:16.043374605 +0100
@@ -16,11 +16,11 @@ endif (GPERFTOOLS_TCMALLOC)
 if(WIN32)
   set(LIBRARIES ${LIBRARIES} "${CMAKE_CURRENT_SOURCE_DIR}/../bamtools/lib/libbamtools.lib" "${CMAKE_CURRENT_SOURCE_DIR}/../win_build/zlibd.lib")
 else(WIN32)
-  set(LIBRARIES ${LIBRARIES} "${CMAKE_CURRENT_SOURCE_DIR}/../bamtools/lib/libbamtools.a" "pthread")
+  set(LIBRARIES ${LIBRARIES} "bamtools" "pthread")
 endif(WIN32)
 
 if (PROTOBUF_FOUND)
-  set(LIBRARIES ${LIBRARIES} "libprotobuf.a")
+  set(LIBRARIES ${LIBRARIES} "protobuf")
 endif(PROTOBUF_FOUND)
 
 target_link_libraries(express ${LIBRARIES})
